test/Microsoft.Extensions.CommandLineUtils.Tests/CommandLineApplicationTests.cs

```diff
@@ -400,7 +400,7 @@ public void HandlesAsyncExecutes()
 
             app.OnExecute(async () =>
             {
-                await Task.Delay(5);
+                await Task.Delay(5).ConfigureAwait(false);
```

Comment:
IMO, these tests are kinda pretty crummy. All the backing code does is call `.Result` on the Task. We could live without these.